modules/light-clients/08-wasm/types/misbehaviour_handle_test.go

```diff
@@ -0,0 +1,195 @@
+package types_test
+
+import (
+	"encoding/json"
+	"errors"
+
+	wasmvm "github.com/CosmWasm/wasmvm"
+	wasmvmtypes "github.com/CosmWasm/wasmvm/types"
+
+	storetypes "cosmossdk.io/store/types"
+
+	wasmtesting "github.com/cosmos/ibc-go/modules/light-clients/08-wasm/testing"
+	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
+	ibcerrors "github.com/cosmos/ibc-go/v8/modules/core/errors"
+	"github.com/cosmos/ibc-go/v8/modules/core/exported"
+	ibctmtypes "github.com/cosmos/ibc-go/v8/modules/light-clients/07-tendermint"
+	ibctesting "github.com/cosmos/ibc-go/v8/testing"
+)
+
+func (suite *TypesTestSuite) TestVerifyClientMessage() {
```

Comment:
super nit: test is in the wrong file (lets do in a followup tho)

Reply:
I moved the test.